requests_cache/cache_keys.py

```diff
@@ -124,7 +124,8 @@ def normalize_request(
 
     norm_request.method = (norm_request.method or '').upper()
     norm_request.url = normalize_url(norm_request.url or '', ignored_parameters)
-    norm_request.headers = normalize_headers(norm_request.headers, ignored_parameters)
+    decoded_headers = {k: decode(v) for (k, v) in norm_request.headers.items()}
```

Comment:
`normalize_headers()` is called in a couple different places, and potentially in other users' custom request matching functions. Could you move this to the first line of `normalize_headers()`?

Reply:
Done in 125d26d.